packages/compass-e2e-tests/helpers/commands/workspace-tabs.ts

```diff
@@ -59,10 +67,15 @@ export async function closeWorkspaceTabs(
       // are multiple tabs then another tab will immediately become active and
       // trip up the logic that checks that the tab you closed went away.
       const id = await currentActiveTab.getAttribute('id');
+      debug('closing tab', { numTabsStart, id });
```

Comment:
Just some debugging in case this still flakes.

Reply:
should we add todos to clean these up later?